app/controllers/admin/users_controller.rb

```diff
@@ -1,5 +1,45 @@
 module Admin
   class UsersController < Admin::ApplicationController
+    USER_PARAMS = %i[
+      name
+      email
+      username
+      twitter_username
+      github_username
+      profile_image
+      website_url
+      summary
+      email_newsletter
+      email_comment_notifications
+      email_follower_notifications
+      organization_id
+      org_admin
+      bg_color_hex
+      text_color_hex
+      employer_name
+      employer_url
+      employment_title
+      currently_learning
+      available_for
+      mostly_work_with
+      currently_hacking_on
+      location
+      email_public
+      education
+      feed_url
+      reputation_modifier
+      saw_onboarding
+      scholar_email
+      facebook_url
+      behance_url
+      dribbble_url
+      medium_url
+      gitlab_url
+      linkedin_url
+      twitch_url
+      instagram_url
+    ].freeze
```

Comment:
This would not work, you cannot freeze something and modify it later.

```
2.5.5 :001 > FROZEN_ARRAY = %i[a b c].freeze
 => [:a, :b, :c]
2.5.5 :002 > FROZEN_ARRAY << %i[d]
Traceback (most recent call last):
        1: from (irb):2
FrozenError (can't modify frozen Array)
```

You either have to create a new array `FROZEN_ARRAY + [:d]` when password is present in the request. Or you have to get rid of the freeze